src/main/java/org/broadinstitute/consent/http/service/DarCollectionService.java

```diff
@@ -560,16 +556,20 @@ public List<DarCollection> addDatasetsToCollections(List<DarCollection> collecti
    */
   public DarCollection cancelDarCollectionAsResearcher(DarCollection collection) {
     Collection<DataAccessRequest> dars = collection.getDars().values();
-    List<String> referenceIds = dars.stream()
-        .map(DataAccessRequest::getReferenceId)
-        .collect(Collectors.toList());
-
-    if (referenceIds.isEmpty()) {
+    if (dars.isEmpty()) {
       logWarn("DAR Collection ID: [%s] does not have any associated DAR ids".formatted(
           collection.getDarCollectionId()));
       return collection;
     }
 
+    DarCollectionSummary summary = darCollectionSummaryDAO
```

Comment:
This change doesn't affect behavior. With this ordering change, if the `dars.isEmpty()` check fails, we don't make an unnecessary DAO call.